src/main/scala/io/iohk/ethereum/utils/Config.scala

```diff
@@ -46,6 +46,14 @@ object Config {
       val waitForChainCheckTimeout = peerConfig.getDuration("wait-for-chain-check-timeout").toMillis.millis
     }
 
+    object Rpc {
```

Comment:
This should probably extend a trait RpcConfig ... and have JsonRpcServer take RpcConfig as a constructor param...